lib/app/modules/teamAttendance/views/team_attendance_view.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import 'package:get/get.dart';
+import 'package:managment_app/Themes/themes.dart';
+
+import '../controllers/team_attendance_controller.dart';
+
+class TeamAttendanceView extends GetView<TeamAttendanceController> {
+
+  @override
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          leading: IconButton(
+            icon: Icon(Icons.arrow_back_rounded),
+            onPressed: () {},
+          ),
+          backgroundColor: Color(0xFF002431),
```

Comment:
use Get.theme.colorLevel1 instead of directly giving hex value i have already assigned the theme use colours according to colorlevel in themes.dart. make changes everywhere similar to this